config/invenio.conf

```diff
@@ -107,7 +107,11 @@ CFG_DATABASE_SLAVE_SU_PASS =
 # i.e. with the username, the 4 characters " // ", and the password.
 CFG_DATABASE_PASSWORD_FILE =
 
-
+# Specify where db dumps are stored locally before being moved to EOS
+CFG_DATABASE_DBDUMP_TEMP_DIR = ''
+# Specify the user that have access to EOS to move db dumps to the EOS directory
+CFG_DATABASE_DBDUMP_EOS_USER = ''
+CFG_DATABASE_DBDUMP_EOS_DIR = ''
```

Comment:
the commit message should be present tens. Also the module name should be `miscutils` instead of `dbdump`.